Maquetado/scss/details.scss

```diff
@@ -13,21 +13,30 @@
   max-height: 415px;
   max-width: 1000px;
   padding: 20px;
+  position: relative;
   width: 100%;
 }
 
 .book-info {
   margin: 0 50px;
+  width: 600px;
```

Comment:
Why this?

Reply:
So the green line can reach the right border